.changeset/weak-adults-scream.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': minor
+---
+
+Added `endIconProps` and `statusMessageProps` to ComboBox. This allows users to customise end icon and status message.
```

Comment:
Typo

```suggestion
Added `endIconProps` and `statusMessageProps` to ComboBox. This allows users to customize end icon and status message.
```

Reply:
"Customize and customise are both English terms."

I guess it's a preference issue.